index.js

```diff
@@ -68,3 +68,27 @@ function validateSudoku(board) {
 }
 
 // trigger validate sudoku through an event handler on the UI
+
+function createBoard() {
```

Comment:
@joeheyming Hey great work!  :) Just a small change. The sudoku board is supposed to be filled by the end user. So this method is not needed. Instead can you add the layout using just HTML and css ? Each of the cell should be a text field which can be editable and can have a maximum length 1 character(number). Thanks!

Reply:
I added the editability of the cells.  Right now it still uses the data structure you provided.  But I'm sure you can change it to loop and fill in the cells with empty values.